src/main/java/scoretag/TagUpdater.java

```diff
@@ -16,10 +14,12 @@ public TagUpdater(Main plugin) {
     public void run() {
         try {
             for (Player p : plugin.getServer().getOnlinePlayers().values()) {
-                String tag = (String) plugin.placeholderAPI.getDeclaredMethod("translateString", String.class, Player.class).invoke(PlaceholderAPI.getInstance(),
-                        plugin.tag.replace("%factions_name%", plugin.getFaction(p)).replace("%device_os%", plugin.getOS(p)), p);
-                if (!p.getScoreTag().equals(tag)) {
-                    p.setScoreTag(tag);
+                String tag = (String) plugin.tag.replace("%factions_name%", plugin.getFaction(p)).replace("%device_os%", plugin.getOS(p));
+                if (Main.api != null) {
```

Comment:
Main.api should never be null

Reply:
I took LuckChat as an example to update the plugin 
https://github.com/ZenSirzechs/LuckChat-NK/blob/c6c173df1a577db25936ab2c9ba65d3b215e37e2/src/main/java/zen/luckchat/NameTag.java#L36